tests/e2e/test_rosacli_upgrade.go

```diff
@@ -269,9 +277,13 @@ var _ = Describe("Cluster Upgrade testing",
 				)
 			}
 			Expect(err).To(BeNil())
-			Expect(output.String()).To(ContainSubstring("are already up-to-date"))
 			Expect(output.String()).To(ContainSubstring("are compatible with upgrade"))
 			Expect(output.String()).To(ContainSubstring("Upgrade successfully scheduled for cluster"))
+			if profile.ClusterConfig.HCP {
```

Comment:
What is the difference between `isHosted` and `profile.ClusterConfig.HCP`. I do not recommend using `profile.xxx`; as I said before, it is an attribute from the template,  but not the real one from the cluster.

Reply:
Thanks. No difference from my side. I will update like you suggest.